runtime/src/cost_model.rs

```diff
@@ -298,19 +309,28 @@ mod tests {
         );
 
         // expected cost for one system transfer instructions
-        let expected_execution_cost = BUILT_IN_INSTRUCTION_COSTS
+        let expected_builtin_cost = *BUILT_IN_INSTRUCTION_COSTS
             .get(&system_program::id())
             .unwrap();
 
-        let mut tx_cost = TransactionCost::default();
-        CostModel::get_transaction_cost(
-            &mut tx_cost,
-            &simple_transaction,
-            &FeatureSet::all_enabled(),
-        );
-        assert_eq!(*expected_execution_cost, tx_cost.builtins_execution_cost);
-        assert_eq!(0, tx_cost.bpf_execution_cost);
-        assert_eq!(3, tx_cost.data_bytes_cost);
+        for feature_set in [FeatureSet::default(), FeatureSet::all_enabled()] {
```

Comment:
is `default()` all disabled? this seems like we're testing more than 1 feature at a time.

Reply:
All features are disabled by `default()`. Yea, the point is valid. I did this because it's easy and (looks) clean - it tests two status: all on or all off, but that may have added ambiguity